src/main/java/frc5124/robot2020/commands/turret/RotateTurret.java

```diff
@@ -5,20 +5,22 @@
 /* the project.                                                               */
 /*----------------------------------------------------------------------------*/
 
-package frc5124.robot2020.commands;
+package frc5124.robot2020.commands.turret;
 
 import edu.wpi.first.wpilibj2.command.CommandBase;
-import frc5124.robot2020.subsystems.Intake;
+import frc5124.robot2020.subsystems.Turret;
 
-public class intakeBalls extends CommandBase {
+public class RotateTurret extends CommandBase {
+  private Turret turret;
+  private double power;
   /**
-   * Creates a new intakeBalls.
+   * Creates a new RotateTurret.
+   * @param power Useable if limit not reached. Suggest moving by units (not coded yet)
    */
-  private Intake m_intake;
-
-  public intakeBalls(Intake subsystem) {
-    m_intake = subsystem;
-    addRequirements(m_intake);
+  public RotateTurret(Turret subsystem, double power) {
```

Comment:
the subsystem should control the speed of the turret rotation. Either we pass in a joystick to the subsystem to allow variable panning/scanning by the operator, or we fix the rate of spin as a constant and simply pass in a direction

Reply:
will set as constant in robotMap, but will leave as varying parameter for versatility. This is done so that we can build upon it in the future depending on what we decide/think. Should the turret always turn at a fixed speed? We should discuss this.